cmd/farmer/main.go

```diff
@@ -18,12 +18,12 @@ import (
 
 var (
 	rootCmd = &cobra.Command{
-		Use:   "farmer",
-		Short: "Farmer",
+		Use:   "storagenode",
```

Comment:
should the filename of this file change?

Reply:
it shows as 

rootCmd = &cobra.Command{
		Use:   "storagenode",
		Short: "StorageNode",
	}

in the PR branch. Can you refresh and check again please.